src/pages/plugins/index.md.ejs

```diff
@@ -2,17 +2,190 @@
+        build_badge:
+          'https://github.com/senecajs/seneca-gateway/actions/workflows/build.yml/badge.svg',
+        coverage_link:
+          'https://coveralls.io/github/senecajs/seneca-gateway?branch=main',
+        coverage_badge:
+          'https://coveralls.io/repos/github/senecajs/seneca-gateway/badge.svg?branch=main',
+        vul_link: 'https://snyk.io/test/github/senecajs/seneca-gateway',
+        vul_badge:
+          'https://snyk.io/test/github/senecajs/seneca-gateway/badge.svg',
+        deepscan_link:
+          'https://deepscan.io/dashboard#view=project&tid=5016&pid=19453&bid=505563',
+        deepscan_badge:
+          'https://deepscan.io/api/teams/5016/projects/19453/branches/505563/badge/grade.svg',
+        maintain_link:
+          'https://codeclimate.com/github/senecajs/seneca-gateway/maintainability',
+        maintain_badge:
+          'https://api.codeclimate.com/v1/badges/9d54b38a991fe7b92a43/maintainability',
+      },
+      {
+        name: 'entity-depend',
+        title: 'entity-depend',
+        desc: 'Handle dependency relationships on entity verison',
+        github_link: 'https://github.com/senecajs/seneca-entity-depend/',
+        npm_link: 'https://www.npmjs.com/package/@seneca/entity-depend',
+        npm_badge: 'https://img.shields.io/npm/v/@seneca/gateway.svg',
+        build_link:
+          'https://github.com/senecajs/seneca-gateway/actions/workflows/build.yml',
+        build_badge:
+          'https://github.com/senecajs/seneca-gateway/actions/workflows/build.yml/badge.svg',
+        coverage_link:
+          'https://coveralls.io/github/senecajs/seneca-gateway?branch=main',
+        coverage_badge:
+          'https://coveralls.io/repos/github/senecajs/seneca-gateway/badge.svg?branch=main',
+        vul_link: 'https://snyk.io/test/github/senecajs/seneca-gateway',
+        vul_badge:
+          'https://snyk.io/test/github/senecajs/seneca-gateway/badge.svg',
+        deepscan_link:
+          'https://deepscan.io/dashboard#view=project&tid=5016&pid=19453&bid=505563',
+        deepscan_badge:
+          'https://deepscan.io/api/teams/5016/projects/19453/branches/505563/badge/grade.svg',
+        maintain_link:
+          'https://codeclimate.com/github/senecajs/seneca-gateway/maintainability',
+        maintain_badge:
+          'https://api.codeclimate.com/v1/badges/9d54b38a991fe7b92a43/maintainability',
+      },
+    ],
+  },
+]
+%>
+
 # Plugins
-*Seneca* plugins enable you to build awesome microservices, incredibly fast.
+
+_Seneca_ plugins enable you to build awesome microservices, incredibly fast.
 Some plugins are maintained in-house, while others are provided by the community.
 
 On this page, we provide an `npm` link and total downloads per month for any plugins
 we find or publish ourselves, so you only need to look in one place.
 
+<!-- Plugin navigation -->
+<%
+let pluginNav = '<table style="table-layout:fixed;width:100%">'
+for (let g = 0; g < pluginsByGroup.length; g++) {
```

Comment:
let's change the data structure a little so it can be more flexible
maps are usually easier to work with than lists as you can have direct references to names things

try:

```
const pluginGroupMap = {
  gateway: {
    '@seneca/gateway-lambda': { ... }
  }
}
```

Then you can use Object.keys to get the list, in order of definition, when you need a list, but you also get direct lookups

Note that I used the full npm name of the plugin as the key for individual plugins - this will be unambiguous, and unique, so also gives us more flexibility, if we want a list of all plugins, say

Reply:
Sounds good, structure updated to remove unnecessary arrays